frontend/Dockerfile

```diff
@@ -1,17 +1,23 @@
-# RedHat UBI 8 with nodejs 14
-FROM registry.access.redhat.com/ubi8/ubi:8.7-1112 AS builder
-RUN dnf module install -y nodejs:18
+# Build static files
+FROM node:alpine3.16 AS build
 
-# Install packages, build and keep only prod packages
 WORKDIR /app
-COPY . ./
-RUN npm ci --omit=dev && npm run build
+COPY . .
+RUN npm ci --omit=dev && npm run build && \
+    npm run build
```

Comment:
Why `npm run build` is being executed twice?

Reply:
Whoops.  Fixed!